omnigibson/action_primitives/curobo.py

```diff
@@ -197,6 +198,16 @@ def __init__(
         for mg in self.mg.values():
             mg.warmup(enable_graph=False, warmup_js_trajopt=False, batch=batch_size)
 
+            # Make sure all cuda graphs have been warmed up
+            for solver in [mg.ik_solver, mg.trajopt_solver, mg.finetune_trajopt_solver]:
+                if solver.solver.use_cuda_graph_metrics:
+                    assert solver.solver.safety_rollout._metrics_cuda_graph_init
+                    if isinstance(solver, lazy.curobo.wrap.reacher.trajopt.TrajOptSolver):
+                        assert solver.interpolate_rollout._metrics_cuda_graph_init
+                for opt in solver.solver.optimizers:
+                    if opt.use_cuda_graph:
+                        assert opt.cu_opt_init
```

Comment:
rubber stamping this, as i have no idea what this does LOL

Reply:
it made sure that the warmup is actually successful (and the cuda graphs are built correctly). Previously it's not successful because our retract_config is wrong.